apps/web/pages/_document.tsx

```diff
@@ -50,9 +50,20 @@ class MyDocument extends Document<Props> {
 
     const nonceParsed = z.string().safeParse(this.props.nonce);
     const nonce = nonceParsed.success ? nonceParsed.data : "";
+
+    const intlLocale = new Intl.Locale(newLocale);
+    // INFO: Typescript does not know about the Intl.Locale textInfo attribute
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-expect-error
```

Comment:
ts-expect-error doesn't need lint bypassing

```suggestion
    // @ts-expect-error INFO: Typescript does not know about the Intl.Locale textInfo attribute
```